build-scratch

```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
-SHA=`git rev-parse HEAD`
-docker build -t $SHA . &&
-    docker run --rm --entrypoint tar $SHA -c /usr/bin/ct Dockerfile.build-scratch | docker build -f Dockerfile.build-scratch --tag coreos/ct -
+
+set -e
+VERSION=$(./git-version)
```

Comment:
Please don't use the git-version script. I thought we got rid of that thing. `git describe` is sufficient.